verification_token/models.py

```diff
@@ -0,0 +1,132 @@
+from datetime import timedelta
+from secrets import token_urlsafe
+
+from django.contrib.auth import get_user_model
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+
+from palvelutarjotin import settings
+
+
+class VerificationTokenManager(models.Manager):
+    def deactivate_token(self, obj_or_class, key=None, verification_type=None):
+        """
+        Deactivate a token. Key parameters is used and must be given
+        when using a model class instead of model instance.
+        """
+        self.filter_active_tokens(obj_or_class, key, verification_type).update(
+            is_active=False
+        )
+
```

Comment:
2 different ways to use this: with an actual model instance or with a class and a key. If an object instance is used, the key-argument is needless and does nothing. If a verification_type is given, only tokens with given type are updated.